bench/README.md

```diff
@@ -65,43 +68,123 @@ And open `http://localhost:9090/` to query and monitor the server-side metrics.
 
 ## Headless
 
-To run write/read load tests
-
-```bash
-locust --headless
-```
-
 To run write load tests
 
-```bash
-locust --headless DCLWriteUser
-```
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
```

Comment:
Mention about `host.docker.internal` hostname when connecting to localhost from locust container. Provide relevant links

Reply:
Done! ✅